aries-site/src/pages/templates/app-layouts.mdx

```diff
@@ -0,0 +1,170 @@
+will be displayed around the perimeter of the content. 
+
+--INSERT-EXAMPLE-HERE--
+
+Use cases include:
+- Dashboards 
+- Tables 
+- Lists
+- Charts
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+#### Narrow
+The "narrow" option scales the content width until a maximum width of `768px`. 
+If the screen is larger than 768, the Page Container will be center aligned 
+and white space will be displayed around the perimeter of the content. 
+
+--INSERT-EXAMPLE-HERE--
+
+Use cases include:
+- Wizards 
+- Articles 
+- Single task flows, such as forms
+- Admin and user settings
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+#### Full
+This option will automatically resize the container to fill the 
+screen displaying it. With the "full" option, if the screen grows wider 
+than the available content, the Page Container will be left aligned to 
+the edge of the screen.
+
+--INSERT-EXAMPLE-HERE--
+
+Use cases include:
+- Charts 
+- Maps 
+- Tables
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+### Grid
+Grommet uses a combination of CSS Grid and Flexbox. CSS Grid 
+allows for the creation of custom grids depending on specific 
+content needs. Items on smaller screens no longer have to stack 
+in the order they are arranged on desktop (unless that is the 
+desired output). 
+
+#### Ordering items based on content needs
+Focusing on content needs can help us create better optimal user 
+experiences &mdash; think about which piece of content a user is likely 
+to interact with first and order items on the page accordingly. 
+
+Specifying layout content in this way will also ensure that it 
+is interepreted appropriately by screen readers for visually 
+impaired users.
+
+--INSERT-EXAMPLE-HERE--
+
+#### Grid Spacing
+Grid allows you to specify row and columns sizes, as well as define 
+margin and gap sizing. The HPE Design System components and templates 
+have grid sizing built in, but when you need them to craft your digital 
+experience, use these values as a reference:
+
+**Row and Column Sizes**  
+
+--PRESENT-THE-FOLLOWING-AS-A-TABLE-OR-MAYBE-EVEN-A-NAMEVALUELIST--
+
+“xxsmall”: “48px”  
+"xsmall": "96px"  
+"small": "192px"  
+"medium": "384px"  
+"large": "768px"  
+"xlarge": "1152px"  
+"xxlarge": "1536px"  
+"full": "100%"  
+
+**Margin, Pad and Gap Sizes**  
+
+--PRESENT-THE-FOLLOWING-AS-A-TABLE-OR-MAYBE-EVEN-A-NAMEVALUELIST--
+
+"none": "0px"  
+"hair": "1px"  
+"xxsmall": "3px"  
+"xsmall": "6px"  
+"small": "12px"  
+"medium": "24px"  
+"large": "48px"  
+"xlarge": "96px"
+
+### Breakpoints
+A breakpoint is a width at which some change in the page design 
+occurs. For designers, breakpoints can help stress test and 
+validate UX decisions around page layouts.
+
+--INSERT-TABLE-HERE--
+
+Margin is the left and right space between the main content of a 
+page and the screen's edge.
```

Comment:
maybe a diagram would be good to add here similar to helps people understand difference between `margin` and `pad` sometimes it can get intermixed 
<img width="236" alt="Screen Shot 2021-12-16 at 6 58 07 PM" src="https://user-images.githubusercontent.com/42451602/146476252-9db59f58-e27a-4b3d-b1cd-5dfb3940e485.png">

Reply:
Nice thought. I think something like this will be useful when we speak to margin, padding, gap and each of their roles. I'd like to see how the rest of the content fills in and where the best place to introduce this concept is. That said, this representation of the box model is still the most effective means for communicating. Glad you called it out.